README.md

```diff
@@ -9,40 +9,78 @@ A module to store and interact with blocks.
 # INSTALL
 `npm install ethereumjs-blockchain`
 
-# API
+# EXAMPLE
+
+The following is an example to iterate through an existing Geth DB (needs ``leveldown`` to be
+installed separately):
+
+```javascript
+const levelup = require('levelup')
+const leveldown = require('leveldown')
+const Blockchain = require('./index.js')
+const utils = require('ethereumjs-util')
+
+var gethDbPath = './chaindata' // Add your own path here
+var db = levelup(gethDbPath, { db: leveldown })
 
-# ethereumjs-blockchain
-A module to store and interact with blocks
+new Blockchain({db: db}).iterator('i', (block, reorg, cb) => {
+  const blockNumber = utils.bufferToInt(block.header.number)
+  const blockHash = block.hash().toString('hex')
+  console.log(`BLOCK ${blockNumber}: ${blockHash}`)
+  cb()
+}, (err) => console.log(err || 'Done.'))
+```
+
+# API
 
 - [`Blockchain`](#blockchain)
     - [`new Blockchain(opts)`](#new-blockchainblockdb-detailsdb)
-    - [`BlockChain` Properties](#blockchain-properties)
     - [`BlockChain` methods](#blockchain-methods)
+        - [`blockchain.putGenesis(genesis, [callback])`](#blockchainputgenesisgenesis-callback)
+        - [`blockchain.getHead(name, [callback])`](#blockchaingetheadname-callback)
+        - [`blockchain.putBlocks(blocks, [callback])`](#blockchainputblocksblocks-callback)
         - [`blockchain.putBlock(block, [callback])`](#blockchainputblockblock-callback)
         - [`blockchain.getBlock(hash, [callback])`](#blockchaingetblockhash-callback)
-        - [`blockchain.getBlockInfo(hash, cb)`](#blockchaingetblockinfohash-cb)
-        - [`blockchain.getBlockHashes(parentHash, count, cb)`](#blockchaingetblockhashesparenthash-count-cb)
-        - [`blockchain.getBlockChain(startingHashes, count, cb)`](#blockchaingetblockhashesparenthash-count-cb)
-        - [`blockchain.selectNeededHashes(hashes, cb)`](#blockchainselectneededhasheshashes-cb)
+        - [`blockchain.getBlocks(blockId, maxBlocks, skip, reverse, [cb])`](#blockchaingetblocksblockid-maxblocks-skip-reverse-cb)
```

Comment:
`[cb]` should be `[callback]` for consistent naming

Reply:
Did this the other way around and now use ``cb`` everywhere since this is also the naming in all the function declarations.